CHANGELOG.md

```diff
@@ -2,3 +2,4 @@
 - Updated Pub/Sub emulator to v0.8.14 (#7197).
 - Updates default resource names and template during `firebase init dataconnect`.
 - Improved handling of 'fresh from Console' services during `init dataconnect`.
+- Update to Firebase Data Connect Emulator v1.2.0, which adds support for Postgres 16 and creates the Postgres database specified in dataconnect.yaml if it does not already exist.
```

Comment:
Shall we describe this as `dataconnect.yaml` or `.firebaserc` or both?

Reply:
Done, mentioned .firebaserc as well.